core/src/main/java/aima/core/search/basic/uninformed/BidirectionalSearch.java

```diff
@@ -25,86 +25,147 @@
  * The strategy of this search implementation is inspired by the description of
  * the bidirectional search algorithm i.e. Problem is reversed and search is performed from both
  * frontiers.
- * @author manthan
+ * @author wormi
```

Comment:
Please don't replace a prior author. If the work is based on, even slightly, on earlier work all authors should remain listed. You may place your name to the head of the list if the most recent changes are predominantly yours.